rkt/api_service.go

```diff
@@ -636,8 +639,54 @@ func (s *v1AlphaAPIServer) InspectImage(ctx context.Context, request *v1alpha.In
 	return &v1alpha.InspectImageResponse{Image: image}, nil
 }
 
+type LogsStreamWriter struct {
+	server v1alpha.PublicAPI_GetLogsServer
+}
+
+func (sw LogsStreamWriter) Write(b []byte) (int, error) {
+	if err := sw.server.SendMsg(b); err != nil {
+		return 0, err
+	}
+	return len(b), nil
+}
+
 func (s *v1AlphaAPIServer) GetLogs(request *v1alpha.GetLogsRequest, server v1alpha.PublicAPI_GetLogsServer) error {
-	return fmt.Errorf("not implemented yet")
+	path := fmt.Sprintf("/var/lib/rkt/pods/run/%s/stage1/rootfs/var/log/journal/", request.PodId)
```

Comment:
Thanks! I'll use those. I figured there would be utility functions for that but still getting more familiar with the codebase.